dgraph-ratel/README.md

```diff
@@ -0,0 +1,15 @@
+# Dgraph Ratel
+
+dgraph-ratel is the front end binary to be used alongside the dgraph distributed graph database.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
```

Comment:
Shouldn't this be a service package?

Reply:
I have mixed feelings on that. It behaves similarly to how the concourse binary behaves. Multiple different services can run from the single binary and it 100% depends upon how you call the binary and what runtime flags you pass it. In these cases I've typically opted to provide the binary and include examples of how to wrap the binary for various contexts. I could be convinced otherwise though :)

Reply:
Yeah potentially this one should be